### PR TITLE
Migrate pkg/kubemark logs to structured logging

### PR DESCRIPTION
**Pull request by wwgfhf**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:
[migration-to-structured-logging](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md)

### CODE REVIEW

File: pkg/kubemark/hollow_kubelet.go [hunk 1]
```diff
@@ -18,6 +18,7 @@ package kubemark
 
 import (
 	"fmt"
+	"k8s.io/utils/mount"
```
Comment (ehashman) [inline pkg/kubemark/hollow_kubelet.go:21]: This new dep doesn't appear to be used anywhere?

**Review by ehashman — COMMENTED** [review]
/sig scalability
/sig instrumentation

File: pkg/kubemark/hollow_proxy.go [hunk 2]
```diff
@@ -82,7 +82,7 @@ func NewHollowProxyOrDie(
 	if useRealProxier {
 		nodeIP := utilnode.GetNodeIP(client, nodeName)
 		if nodeIP == nil {
-			klog.V(0).Infof("can't determine this node's IP, assuming 127.0.0.1")
+			klog.V(0).InfoS("Can't determine this node's IP, assuming 127.0.0.1")
```
Comment (mm4tt) [inline pkg/kubemark/hollow_proxy.go:85]: Why InfoS if there are no key values?

File: pkg/kubemark/hollow_kubelet.go (hunk 1 above)
Comment (mm4tt) [reply pkg/kubemark/hollow_kubelet.go:21]: +1, I believe lint will catch this

**Review by mm4tt — COMMENTED** [review]
/ok-to-test

File: pkg/kubemark/hollow_kubelet.go [hunk 3]
```diff
@@ -153,7 +154,7 @@ type HollowKubletOptions struct {
 func GetHollowKubeletConfig(opt *HollowKubletOptions) (*options.KubeletFlags, *kubeletconfig.KubeletConfiguration) {
 	testRootDir := utils.MakeTempDirOrDie("hollow-kubelet.", "")
 	podFilePath := utils.MakeTempDirOrDie("static-pods", testRootDir)
-	klog.Infof("Using %s as root dir for hollow-kubelet", testRootDir)
+	klog.InfoS("Using testRootDir as root dir for hollow-kubelet", "testRootDir", testRootDir)
```
Comment (serathius) [inline pkg/kubemark/hollow_kubelet.go:157]:
```suggestion
	klog.InfoS("Using testRootDir as root dir for hollow-kubelet", "path", testRootDir)
```

File: pkg/kubemark/hollow_proxy.go (hunk 2 above)
Comment (serathius) [reply pkg/kubemark/hollow_proxy.go:85]: We can mark directory as migrated in static analysis.

**Comment by dims** [comment]
this PR has been on hold for about 2 weeks. Can you please see how to unblock this by reviewing comments etc? or please close this out (Trying to declutter our open list of PRs)

**Review by yangjunmyfm192085 — COMMENTED** [review]
/cc @wwgfhf, are you still working on this pr?

**Comment by pchan** [comment]
This issue was discussed in the structured logging WG, I will open a new PR based on this one.

**Pull request closed without merging**